src/ontology/mondo-edit.obo

```diff
@@ -481911,14 +481911,14 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/888/biliary-
 
 [Term]
 id: MONDO:0022575
-name: biliary hypoplasia
-def: "A syndromic disease characterized by a small ductal system and reduction in the number of interlobular bile ducts." [https://orcid.org/0000-0001-5208-3432, PMID:12664410]
+name: obsolete biliary hypoplasia
+def: "OBSOLETE. A syndromic disease characterized by a small ductal system and reduction in the number of interlobular bile ducts." [https://orcid.org/0000-0001-5208-3432, PMID:12664410]
 comment: Editor note: consider making this a finding; Reason of obsoletion: is a phenotype and not a disease - MONDO:excludePhenotype. Term to consider: -
 subset: obsoletion_candidate
```

Comment:
@yshwetar make sure this is removed. After obsoletion, this term is not an obsoletion candidate anymore

Reply:
Got it! Thank you